react/components/ProductAvailabilityWrapper.tsx

```diff
@@ -131,6 +163,16 @@ ProductAvailabilityWrapper.schema = {
       description: messages.highStockMessageDescription.id,
       type: 'string',
     },
+    showAvailability: {
+      title: messages.showAvailabilityTitle.id,
+      description: messages.showAvailabilityDescription.id,
+      type: 'string',
```

Comment:
Giving it type `string` makes users think it accepts any stock value, which is not true. You should turn it into an enum. Look at this [example](https://github.com/vtex-apps/store-image/blob/7ba598a7fd54ba050b4a53510dfac2b64db1332c/store/contentSchemas.json#L32).

```suggestion
      type: 'enum',
```

Reply:
I took a look at the example, however the structure is different. Besides turning type from string to enum, should i include the enum options in an enum field, like this?

enum: ['disabled', stock]

@icazevedo

Reply:
Exactly @favri

Reply:
Thanks @icazevedo , can you give a final look to the component please?